vector/src/main/java/im/vector/home/BaseNewHomeIndividualFragment.kt

```diff
@@ -0,0 +1,90 @@
+package im.vector.home
+
+import android.content.Context
+import android.os.Bundle
+import android.util.Log
+import android.view.View.GONE
+import androidx.core.content.ContextCompat
+import im.vector.R
+import im.vector.adapters.AbsAdapter
+import im.vector.adapters.HomeRoomAdapter
+import im.vector.fragments.AbsHomeFragment
+import im.vector.ui.themes.ThemeUtils
+import kotlinx.android.synthetic.main.fragment_home_individual.*
+import org.matrix.androidsdk.data.Room
+import java.util.*
+import kotlin.collections.ArrayList
+
+
+abstract class BaseNewHomeIndividualFragment : AbsHomeFragment(), UpDateListener {
+    private val LOG_TAG = BaseNewHomeIndividualFragment::class.java.simpleName
+
+    var registerListener: RegisterListener? = null
+    var onSelectRoomListener: HomeRoomAdapter.OnSelectRoomListener? = null
+    var invitationListener: AbsAdapter.RoomInvitationListener? = null
+    var moreActionListener: AbsAdapter.MoreRoomActionListener? = null
+
+    val localRooms = ArrayList<Room>()
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+        activity?.let { activity ->
+            mPrimaryColor = ThemeUtils.getColor(activity, R.attr.vctr_tab_home)
+            mSecondaryColor = ThemeUtils.getColor(activity, R.attr.vctr_tab_home_secondary)
+            mFabColor = ContextCompat.getColor(activity, R.color.tab_rooms)
```

Comment:
Should mFabColor be mTabColor ?

Reply:
Fab stands for the floating action button and this has been defined in the parent class (original Riot).

Reply:
tab_color is using accent color. The same code is used in the original HomeFragment.